paparazzi/src/main/java/app/cash/paparazzi/internal/resources/RepositoryLoader.kt

```diff
@@ -927,15 +927,12 @@ abstract class RepositoryLoader<T : LoadableResourceRepository>(
   }
 
   protected fun getResRelativePath(file: PathString): String {
-    if (file.isAbsolute) {
+    if (file.isAbsolute || !isLoadingFromZipArchive) {
       return resourceDirectoryOrFilePath.relativize(file).portablePath
     }
     assert(file.nameCount != 0)
```

Comment:
since we're reverting back to the Android Studio way, let's include this comment.  for reference: https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:resource-repository/main/java/com/android/resources/base/RepositoryLoader.java;l=125?q=RepositoryLoader&sq=

```suggestion
    // The path is already relative, drop the first "res" segment.
    assert(file.nameCount != 0)
```